### PR TITLE
Updates for VBF limit results

### PR DESCRIPTION
**Pull request by alpakpinar**
- Here, for compatibility with the histogram naming at the legacy_vbf.py file, EWK_V merging is removed. Instead, EWK_WMinus and EWK_WPlus samples are merged to EWK_W and EWK_Z samples are untouched.
- The histogram naming scheme is updated accordingly, in legacy_vbf.py .

### CODE REVIEW

File: bucoffea/plot/util.py [hunk 1]
```diff
@@ -207,6 +207,10 @@ def merge_datasets(histogram):
         'WNJetsToLNu-MLM_2017' : [x for x in all_datasets if re.match('W(\d+)JetsToLNu_2017',x)],
         'WNJetsToLNu-MLM_2018' : [x for x in all_datasets if re.match('W(\d+)JetsToLNu_2018',x)],
 
+        'Diboson_2017' : [x for x in all_datasets if re.match('(WW|WZ|ZZ|WW).*_2017',x)],
```
Comment (AndreasAlbert) [inline bucoffea/plot/util.py]: Isnt that already [here](https://github.com/alpakpinar/bucoffea/blob/2020-01-10_remove_ewk_merging/bucoffea/plot/util.py#L234) ?

File: bucoffea/plot/util.py [hunk 2]
```diff
@@ -207,6 +207,10 @@ def merge_datasets(histogram):
...
+        'Diboson_2018' : [x for x in all_datasets if re.match('(WW|WZ|ZZ|WW).*_2018',x)],
+        'EWKW2Jets_WToLNu_M-50-mg_2017' : [x for x in all_datasets if re.match('EWKW(Plus|Minus)2Jets.*-mg_2017',x)],
+        'EWKW2Jets_WToLNu_M-50-mg_2018' : [x for x in all_datasets if re.match('EWKW(Plus|Minus)2Jets.*-mg_2018',x)],
```
Comment (AndreasAlbert) [inline bucoffea/plot/util.py]: Can this also be put into the yearly loop down below? Should be the default

**Review by AndreasAlbert — CHANGES_REQUESTED** [review]

File: bucoffea/plot/util.py (hunk 1 above)
Comment (alpakpinar) [reply bucoffea/plot/util.py]: You're right, there was a merge conflict and while fixing it I think I got some duplicates, I'll fix it.

File: bucoffea/plot/util.py (hunk 2 above)
Comment (alpakpinar) [reply bucoffea/plot/util.py]: Sure, I can put that in the yearly loop too.

File: bucoffea/plot/util.py (hunk 1 above)
Comment (alpakpinar) [reply bucoffea/plot/util.py]: Done in commit: https://github.com/alpakpinar/bucoffea/commit/9cdf54adb2cdf2f5d7a313f35e040331d65e3743

File: bucoffea/plot/util.py (hunk 2 above)
Comment (alpakpinar) [reply bucoffea/plot/util.py]: Done in commit: https://github.com/alpakpinar/bucoffea/commit/dcf0dee0db3381b8cd0faa49bc3eded217a13a21

**Review by AndreasAlbert — APPROVED** [review]

**Pull request merged**